web/hooks/github.go

```diff
@@ -167,6 +168,7 @@ func (wh GitHubWebHook) runAssignmentTests(assignment *pb.Assignment, repo *pb.R
 		JobOwner:   payload.GetSender().GetLogin(),
 	}
 	if assignment.SkipTests {
+		wh.logger.Debugf("Assignment %s for course %s has no automated test, skipping tests", assignment.Name, course.Name)
```

Comment:
Suggested phrasing: `Assignment %s for course %s is manually reviewed.`

Reply:
Done.